flit-server/test/user.test.ts

```diff
@@ -10,12 +10,13 @@ describe("GET /login", () => {
 });
 
 
-describe("GET /forgot", () => {
-    it("should return 200 OK", () => {
-        return request(app).get("/forgot")
-            .expect(200);
-    });
-});
+// This code path is temporarily disabled.
```

Comment:
I'd like to see comments on this and the other "temporarily disabled" clarifying when they will be re-enabled so we don't have commented-out code without a clear idea of its lifespan.

Reply:
Good call. Updated.